tests/test_fips.py

```diff
@@ -320,3 +320,35 @@ def test_icainfo_binary(container_per_test: ContainerData) -> None:
     )
     container_per_test.connection.check_output("icastats -k")
     container_per_test.connection.check_output("icastats -S")
+
+
+@pytest.mark.parametrize(
+    "container_per_test", FIPS_TESTER_IMAGES, indirect=True
+)
+def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
+    container_per_test.connection.check_output(
+        "zypper -n install mozilla-nss mozilla-nss-tools"
+    )
+    # Seed file
+    container_per_test.connection.check_output(
+        "cat /dev/urandom | head -n 120 > seedfile.dat"
```

Comment:
this is a really awkward command since head is using text formatting (counting lines) and urandom is binary data. 

how about using "dd if=/dev/urandom bs=1k count=XXX of=seedfile.dat" instead (where XXX is a sensible value that we need?)

Reply:
Had the same thought, but are we sure `dd` is installed by default everywhere?

Reply:
I was just following the steps in the [Confluence page](https://confluence.suse.com/display/qasle/15-SP6+FIPS+Vendor+Affirmation) which are the same as in the [GitHub issue](https://github.com/SUSE/BCI-tests/issues/615), so I guess there's room to improve them

Reply:
Changed :heavy_check_mark: